index.d.ts

```diff
@@ -1155,6 +1155,13 @@ declare namespace plugins {
      * The service name to be used for this plugin. If a function is used, it will be passed the connection parameters and its return value will be used as the service name.
      */
     service?: string | ((params: any) => string);
+    /**
+     * Whether to include query parameters. By default this is disabled and only query text with question marks will be logged.
```

Comment:
> By default this is disabled and only query text with question marks will be logged.

Not sure I understand this part since by default nothing will be logged related to the query parameters.

Reply:
Also, there should be a quick explanation of how to use the function, especially for non-TypeScript users that can't rely on the types to prevent misuse. Speaking of TypeScript, are the params actually `any` or would they always be strings?

Reply:
If you use params for a query, the query needs to have question marks where the values of the params will be inserted. I removed it to not cause confusion. I also added an explanation of the callback.